src/config/Strings.js

```diff
@@ -68,4 +68,15 @@ export const Strings = {
     mainTitle: 'Ne găsești aici',
     backgroundTitle: 'Contact',
   },
+  footer: {
+    copyrightTitle: 'Drepturi de autor © 2023',
+    owner: 'Școala Auto CLD',
```

Comment:
Change this to "CLD Auto Drive Consult SRL". Scoala Auto CLD is the comercial name and I think the actual ltd name is more suitable here.

Reply:
Replaced string as specified
<img width="550" alt="Screenshot 2023-02-24 at 12 51 08" src="https://user-images.githubusercontent.com/37547839/221161322-10ec6223-b1a2-47a3-ad29-e9616ad0037e.png">